.github/workflows/main.yml

```diff
@@ -28,10 +28,10 @@ jobs:
         run: isort . --check
 
   Tests:
-    runs-on: ubuntu-latest
```

Comment:
The latest ubuntu does not support **Python 3.6**
https://github.com/actions/setup-python/issues/544